pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = --cov . --cov-config .coveragerc --pep8 --pylint --cov-report term --cov-report html --ds=ccxcon.settings
+pep8ignore =
```

Comment:
Can we get an opinion from @bdero on line length settings? He's had to suffer through the most diffs under duress.